### PR TITLE
Statement of faith

### PR DESCRIPTION
**Pull request by jakecatron**

### CODE REVIEW

File: statementoffaith.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+---
+
+<section class="jumbotron jumbo-height" style="background-image: url('//crds-media.imgix.net//23IWaWUAICoAUG4Jassq40/24e97abb4beedcc1e41c334e06d4305e/crossroads-church-old2.png'); height: 455px;">
+    <h1 class="component-header-box component-header-box-border-lg mt-100">
```
Comment (tcmacdonald) [inline statementoffaith.html:6]: `mt-100` is not a thing. Can you please remove? Thanks!

File: statementoffaith.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+---
+
+<section class="jumbotron jumbo-height" style="background-image: url('//crds-media.imgix.net//23IWaWUAICoAUG4Jassq40/24e97abb4beedcc1e41c334e06d4305e/crossroads-church-old2.png'); height: 455px;">
```
Comment (silasbaur) [inline statementoffaith.html:5]: What is `jumbo-height` doing?

Comment (jakecatron) [reply statementoffaith.html:5]: ah, thanks for catching that. It was an old class not being used anymore.

File: statementoffaith.html (hunk 1 above)
Comment (jakecatron) [reply statementoffaith.html:6]: Sorry about this, I should've left a note that it was on the our-history branch.

**Comment by tcmacdonald** [comment]
Looks good @jakecatron – should this page have a header / footer?

**Pull request merged**